dev/com.ibm.ws.ejbcontainer.mdb/resources/com/ibm/ws/ejbcontainer/mdb/resources/EJBMDBMessages.nlsprops

```diff
@@ -66,3 +66,11 @@ MDB_ACTIVATION_SPEC_NOT_FOUND_CNTR4015W.useraction=Verify that an activation spe
 MDB_DESTINATION_NOT_FOUND_CNTR4016W=CNTR4016W: The message endpoint for the {0} message-driven bean cannot be activated because the {1} destination is not available. The message endpoint will not receive messages until the destination becomes available.
 MDB_DESTINATION_NOT_FOUND_CNTR4016W.explanation=The server is starting the message-driven bean but is unable to activate the message endpoint because the destination could not be found. The message endpoint will not receive messages until the endpoint can be activated.
 MDB_DESTINATION_NOT_FOUND_CNTR4016W.useraction=Verify that a destination has been configured with the specified name. If the configuration exists, wait for the destination to become available.
+
+MDB_ENDPOINT_ALREADY_INACTIVE_CNTR4117I=CNTR4117I: The message endpoint for the {0} message-driven bean was not paused because it was already in an inactive state.
```

Comment:
The ID reviewer will provide better comments here, but to me it sounds a bit awkward to say,
... message-driven bean was not paused because it was already in an inactive state.
One idea for rewording it:
... message-driven bean cannot be paused because it is not active.